reference/README.md

```diff
@@ -100,7 +100,7 @@ This is a list of Concepts for which an exercise has been implemented:
 | `nested-types`                                             | [C#](../languages/csharp/exercises/concept/nested-types/.docs/instructions.md), [C#](../languages/csharp/exercises/concept/nested-types/.docs/instructions.md)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
-| `nil`                                                      | [Elixir](../languages/elixir/exercises/concept/nil/.docs/instructions.md), [Go](../languages/go/exercises/concept/nil/.docs/instructions.md), [Ruby](../languages/ruby/exercises/concept/instance-variables/.docs/instructions.md)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `nil`                                                      | [Elixir](../languages/elixir/exercises/concept/nil/.docs/instructions.md), [Ruby](../languages/ruby/exercises/concept/instance-variables/.docs/instructions.md)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
@ErikSchierboom: `reference/README.md` should not be checked by the link checker, since it is autogenerated. This outdated link blocked my PR. (renamed the exercise)

Reply:
@tehsphinx Hmmm, I'd rather not exclude this file from the link checker, as it is quite important to have the links be correct in this file. I can see where renaming an exercise is problematic, but I'd argue that due to that being relatively uncommon, I'd prefer to keep the link checker.

In cases where an exercise is renamed, maybe we should allow the PR submitter to edit the generated file, just to have the build pass.